### PR TITLE
Add NL model

### PR DESCRIPTION
**Pull request by Pitchiu**
This PR adds NL: model, metadata and descriptions.

### CODE REVIEW

File: model/countries/NL/NL-descriptions.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  name:
+    en: Full name
+    nl: Naam en achternaam
+  honorific-prefix:
+    en: Honorific prefix
+    nl: Aanhef
+  given-name:
+    en: Given name
+    nl: Voornaam
+# family-name-prefix:
```
Comment (Pitchiu) [inline model/countries/NL/NL-descriptions.yaml:11]: Name handling for NL is still under consideration.

**Comment by skyman503** [comment]
The model LGTM :)

**Review by skyman503 — APPROVED** [review]

File: model/countries/NL/NL-model.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+cut-off-children:
+  - street
+  - building
+  - unit
+
+cut-off-tokens:
+  - organization-title
```
Comment (norgevz) [inline model/countries/NL/NL-model.yaml:7]: Let's also include  `honorific-suffix` as part of `cut-off-tokens`.

**Review by norgevz — APPROVED** [review]
LGTM % one ask.

Well done!

File: model/countries/NL/NL-model.yaml (hunk 2 above)
Comment (Pitchiu) [reply model/countries/NL/NL-model.yaml:8]: Done, thanks!

**Pull request merged**